terraform/gcp/README.md

```diff
@@ -0,0 +1,2 @@
+# Terraform Cloud GCP CI/CD
```

Comment:
@diego3158 in case you're unfamiliar with markdown, this is a readme file so that when you land on this gcp terraform folder, you get a sort of formatted web page. but you can name them whatever you want and they're really useful documenting all sorts of things. but README.md is the default because github assumes it's a sort of web index page.